users/factories.py

```diff
@@ -15,6 +15,7 @@ class UserFactory(DjangoModelFactory):
     username = FuzzyText()
     email = FuzzyText(suffix="@example.com")
     name = Faker("name")
+    password = FuzzyText()
```

Comment:
This might hit a length constraint

Reply:
I am going to remove it from here as the password is no more required for email change.